api/_mixins/ParentNode__Element.json

```diff
@@ -430,59 +192,11 @@
             "standard_track": true,
             "deprecated": false
           }
-        },
-        "document_documentfragment": {
-          "__compat": {
-            "description": "Support on <code>Document</code> and <code>DocumentFragment</code>",
-            "support": {
-              "chrome": {
-                "version_added": "29"
-              },
-              "chrome_android": {
-                "version_added": true
-              },
-              "edge": {
-                "version_added": "≤79"
-              },
-              "firefox": {
-                "version_added": "25"
-              },
-              "firefox_android": {
-                "version_added": null
-              },
-              "ie": {
-                "version_added": false
-              },
-              "opera": {
-                "version_added": "16"
-              },
-              "opera_android": {
-                "version_added": null
-              },
-              "safari": {
-                "version_added": false
-              },
-              "safari_ios": {
-                "version_added": false
-              },
-              "samsunginternet_android": {
-                "version_added": true
-              },
-              "webview_android": {
-                "version_added": "≤37"
-              }
-            },
-            "status": {
-              "experimental": false,
-              "standard_track": true,
-              "deprecated": false
-            }
-          }
         }
       },
       "lastElementChild": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/ParentNode/lastElementChild",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/lastElementChild",
```

Comment:
Things to update outside of the diff:
- chrome_android should be "18" based on mirroring
- opera_android should most likely be "10.1" (the first Opera for Android release) as in the other entries
- safari_ios "3" based on https://trac.webkit.org/changeset/34452/webkit
- samsunginternet_android should be "1.0" based on mirroring